pyaedt/modeler/cad/Modeler.py

```diff
@@ -329,6 +372,36 @@ def rename(self, newname):
         self.name = newname
         return True
 
+    @pyaedt_function_handler()
+    def delete(self):
+        """Delete the coordinate system.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+
+        Examples
+        --------
+        Clean all coordinate systems of the design.
```

Comment:
```suggestion
        Clean all coordinate systems in the design.
```
Should "Clean" be changed to "Delete" or "Clear"?